Runtime/Interaction/InteractionSimulatorDummy.cs

```diff
@@ -0,0 +1,46 @@
+﻿using Innoactive.Creator.BasicInteraction;
+using UnityEngine;
+
+namespace Innoactive.Creator.BasicInteraction
+{
+    public class BaseInteractionSimulatorDummy : BaseInteractionSimulator
+    {
+        private const string ErrorMessage = "You are using the interaction simulator without providing a concrete implementation of it your VR interaction framework.";
```

Comment:
```suggestion
        private const string ErrorMessage = "You are using the interaction simulator without providing a concrete implementation of your XR interaction framework.";
```

Reply:
This is not about the unity XRI, it's about any VR interaction framework.